e2e/playwright/fixtures/cmdBarFixture.ts

```diff
@@ -151,4 +151,11 @@ export class CmdBarFixture {
   chooseCommand = async (commandName: string) => {
     await this.cmdOptions.getByText(commandName).click()
   }
+
+  /**
+   * Select an option from the command bar
+   */
+  selectOption = (options: Parameters<typeof this.page.getByRole>[1]) => {
```

Comment:
What a funky type :)

Reply:
I'm just curious what does VS Code show for the intellisense?

Reply:
Yeah haha is this not a good idea? Here's intellisense:

![Screenshot 2025-01-21 at 7 43 59 PM](https://github.com/user-attachments/assets/21dc2c7a-d0d7-40a1-b47c-1452a3792778)